pkg/utils/client/apply_test.go

```diff
@@ -188,4 +197,33 @@ var _ = Describe("Applier Suite", func() {
 			})
 		})
 	})
+
+	Describe("with CRDs", func() {
+		var foo *unstructured.Unstructured
+		var crd *apiextensionsv1beta1.CustomResourceDefinition
+
+		BeforeEach(func() {
+			foo = test.ExampleFoo.DeepCopy()
+			crd = test.ExampleCRD.DeepCopy()
+		})
+
+		Describe("with the CRD not installed", func() {
```

Comment:
Nitpick; what do you think about making this "when the CRD is not installed"? And similar for the one below when it is installed.

Reply:
I've made this change, it reads better, thanks!